django_freeradius/base/admin.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.admin import ModelAdmin
+
+
+class TimeStampedEditableAdmin(ModelAdmin):
+    """
+    ModelAdmin for TimeStampedEditableModel
+    """
+    def __init__(self, *args, **kwargs):
+        self.readonly_fields += ('created', 'modified',)
```

Comment:
this can be better done with ``ModelAdmin.get_readonly_fields`` https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#django.contrib.admin.ModelAdmin.get_readonly_fields